api/src/pcapi/routes/backoffice/templates/chronicles/list.html

```diff
@@ -66,6 +67,7 @@ <h2 class="fw-light">Chroniques</h2>
                 <td class="pc-mw-50-vw">{{ collapsable_text(chronicle.id, chronicle.short_content|trim, chronicle.content|trim) }}</td>
                 <td>{{ chronicle.dateCreated  | format_date }}</td>
                 <td>{{ chronicle.isActive | format_bool_badge }}</td>
+                <td>{{ chronicle.isSocialMediaDiffusible| format_bool_badge }}</td>
```

Comment:
Anecdotique : Parce qu'il faut bien trouver quelque chose, et que je suis plus pointilleux que le linter :smile: 
```suggestion
                <td>{{ chronicle.isSocialMediaDiffusible | format_bool_badge }}</td>
```